bundles/ch.elexis.core.ui.contacts/src/ch/elexis/core/ui/contacts/views/PatientDetailView.java

```diff
@@ -263,10 +265,10 @@ public String getLabel(Object o) {
 					Kontakt other = Kontakt.load(bezugsKontakt.get(BezugsKontakt.OTHER_ID));
 					if (other.exists()) {
 						List<String> tokens = new ArrayList<String>();
+						String mobile = other.get(Kontakt.FLD_MOBILEPHONE);
 
 						String telefon1 = other.get(Kontakt.FLD_PHONE1);
 						String telefon2 = other.get(Kontakt.FLD_PHONE2);
-						String mobile = other.get(Kontakt.FLD_MOBILEPHONE);
 						String eMail = other.get(Kontakt.FLD_E_MAIL);
```

Comment:
diesen patch brauchts doch nicht? Die Reihenfolge in der du die variablen ausliest hat ja
keine Auswirkung auf eine eventuelle Reihung irgendwo oder?

Reply:
ich habe dies lediglich für die ersichtliche ordnung geändert, damit wenns im elexis so geordnet ist überall sonst auch geordnet ist, enthält aber keine weiteren Auswirkungen